WooCommerce/Classes/View Modifiers/View+AutofocusTextModifier.swift

```diff
@@ -27,14 +26,6 @@ extension View {
     /// Autofocus in `TextField` and `TextEditor` is available only for iOS15+
     ///
     func focused() -> some View {
-        // Conditional check has to be done inside the Group function builder,
-        // otherwise the iOS 15 modifier will be loaded into memory and the app will crash.
-        Group {
-            if #available(iOS 15.0, *) {
-                self.modifier(AutofocusTextModifier())
-            } else {
-                self
-            }
-        }
+        modifier(AutofocusTextModifier())
```

Comment:
I explicitly checked this still worked:

<img alt="focused" src="https://user-images.githubusercontent.com/1218433/205268616-6074652c-473a-4dfa-a034-eccf787b2cd4.gif" width=300/>